jobs/pr_gate/docker/monorail/config.json

```diff
@@ -0,0 +1,64 @@
+{
```

Comment:
@hohene Thanks!
I just checked the "test/config/rackhd_defaults.json"
It wrapped the config.json of RackHD with ``` "rackhd-config"```.
So, if I copy the file config.json to test/config/rackhd_default.json, the FIT will failed to read ```rackhd-config```.
Currently, it's ok not to replace the config.json.
Because FIT only used the "amqp" from "test/config/rackhd_default.json" and the value is the same as jobs/pr_gate/docker/monorail/config.json.

I suggest we remove the wrapper from ```test/config/rackhd_defaults.json```